releases.md

```diff
@@ -0,0 +1,889 @@
+# Mobile Release Notes
+
+###### tags: `mobile` `core`
+
+# Release checklist
+
+#### Before submission
+- [ ] Feature or improvements release?
+- [ ] Sign off by QA
+- [ ] App store listing content reviewed and updated ([copy](https://docs.google.com/spreadsheets/d/1fLDr3BD1dcg0dolvLiozXoZU27IQirjBzn0h4_QS2u0/edit#gid=0), [images](https://www.dropbox.com/sh/6baklctvxy3nuoo/AACG9kYIVdt2qREW76rP8SP-a?dl=0) )
+- [ ] Play store content reviewed and updated
+- [ ] Release notes written and added (500 character limit for updates (Android), no Android references for App store)
+- [ ] Comms timeframe agreed on (Mon/Tue ping when both submissions are in)
+- [ ] Privacy policy reviewed and updated
+- [ ] App translations for key features merged and tested
+- [ ] Countries to be excluded
+
+#### Before publishing
+- [ ] Translations for comms available
+- [ ] Promotional content ready
+- [ ] Update Draft with release on https://github.com/status-im/status-react/releases
+- [ ] Add release to DO store and [update site](https://notes.status.im/s/status-release-upload#) (Jakub - requires credentials to upload) 
```

Comment:
This checklist shouldn't be here. I'd move it to a separate file. Like `doc/RELEASE_CHECKLIST.md` or something.

Reply:
agreed, will move to a separate file